### ISSUE TITLE
"lose" spelled as "loose"

### ISSUE THREAD

**Issue by herolsen** [duplicati/duplicati#2860]
- [ ] I have searched open and closed issues for duplicates.

----------------------------------------

## Environment info

- **Duplicati version**: 2.0.2.1
- **Operating system**: Windows
- **Backend**: SFTP

## Description

Spelling error -- 

```
$ git diff
diff --git a/Localizations/webroot/localization_webroot.pot b/Localizations/webroot/localization_webroot.pot
index c881620..4c4f8aa 100644
--- a/Localizations/webroot/localization_webroot.pot
+++ b/Localizations/webroot/localization_webroot.pot
@@ -2512,7 +2512,7 @@ msgid "You have chosen to restore to a new location, but not entered one"
 msgstr ""
 
 #: scripts/controllers/EditBackupController.js:308
-msgid "You have generated a strong passphrase. Make sure you have made a safe copy of the passphrase, as the data cannot be recovered if you loose the passphrase."
+msgid "You have generated a strong passphrase. Make sure you have made a safe copy of the passphrase, as the data cannot be recovered if you lose the passphrase."
 msgstr ""
 
 #: scripts/controllers/EditBackupController.js:238

$ git commit -m "correct spelling -- lose (to misplace), not loose (not attached)"
```

## Steps to reproduce

1. Add backup
2. Choose auto-generated (strong) passphrase
3. Get to reminder at end not to forget ...

- **Actual result**:
loose
- **Expected result**:
lose

**Comment by davegold24** [comment]
@kenkendk I did a pull request and assigned it to merge for Duplicati/Library/Main/Utility.cs and Duplicati/Server/webroot/ngax/scripts/controllers/EditBackupController.js
Once you've verified that I did those properly, I'll do the localizations that also have this issue.

**Comment by davegold24** [comment]
localizations all updated, pending @kenkendk to verify and commit.

**Comment by davegold24** [comment]
@herolsen I'm not sure what release version will have the spelling fixed in, but as soon as they are committed, I'll close this issue.

**Comment by kenkendk** [comment]
Fixed with c3fe4bbe515bac6a85c8e560484ce0b9626246e0
